templates/slo-rules.jsonnet.j2

```diff
@@ -0,0 +1,81 @@
+local slo = import 'valet-libsonnet/valet.libsonnet';
+
+// Labels that will be reused in SLO rules
+local labels = [{{ labels_to_selectors(load_json(labels)) }}];
+
+{%- for h in http_rates %}
+local {{ h.name + "__http_rates" }} = slo.httpRates({
+  metric: '{{ h.metric }}',
+  selectors: [{{ labels_to_selectors(load_json(h.selectors)) }}],
+  labels: labels,
+  {%- if h.httpStatusLabel %}
+  codeSelector: h.httpStatusLabel,
+  {%- endif %}
+});
+{%- endfor %}
+
+{%- for l in latency_rates %}
+local {{ l.name + "__latency_rates" }} = slo.latencyPercentileRates({
+  metric: '{{ l.metric }}',
+  selectors: [{{ labels_to_selectors(load_json(l.selectors)) }}],
+  percentile: '{{ l.percentile }}',
+  labels: labels,
+});
+{%- endfor %}
+
+{%- for v in volume %}
+local {{ v.name + '__volume' }} = slo.volumeSLO({
+  rules: {{ v.rule + '__http_rates.rateRules' }},
+  threshold: '{{ v.threshold }}',
+  {%- if v.additional_labels %}
+  labels: {{ labels_to_selectors(v.additional_labels) }},
+  {%- endif %}
+});
+{%- endfor %}
+
+{%- for l in latency %}
+local {{ l.name + '__latency' }} = slo.latencySLO({
+  rules: {{ l.rule + '__latency_rates.rules' }},
+  threshold: '{{ l.threshold }}',
+  {%- if l.additional_labels %}
+  labels: {{ labels_to_selectors(l.additional_labels) }},
+  {%- endif %}
+});
+{%- endfor %}
+
+{%- for e in errors %}
+local {{ e.name + '__errors' }} = slo.errorsSLO({
+  rules: {{ e.rule + '__http_rates.errorRateRules' }},
+  threshold: '{{ 100 - e.target | int }}',
+  {%- if e.additional_labels %}
+  labels: {{ labels_to_selectors(e.additional_labels) }},
+  {%- endif %}
+});
+{%- endfor %}
+
+{%- for a in availability %}
+local {{ a.name + '__availability' }} = slo.availabilitySLO({
```

Comment:
Is this correct? availabilitySLO expects two args: https://github.com/rporres/valet-libsonnet/blob/master/slos.libsonnet#L68

Reply:
my mistake, I was using the wrong version of valet-libsonnet

Reply:
the signature of that function has changed in the PR version: https://github.com/rporres/valet-libsonnet/blob/simplify-handlers-support/slos.libsonnet#L79